### ISSUE TITLE
add led strips inside the laser cutter

### ISSUE THREAD

**Issue by jackie1050** [DoESLiverpool/somebody-should#376]
Rob Eaton's suggestion, but he says he doesn't do social media ...

**Comment by DefProc** [comment]
The current light inside is a fluorescent tube, but the tube has died. This could be fixed by just changing the tube, although I'd agree that LEDs should reduced the need for future changes. 

There's mains supply for the current light inside the cutting volume, so mains led lights would be simpler. If we choose low voltage strips with the a power supply, we would be better taking the mains feed from the switch on the control panel, so the AC-DC converter is kept out of the smoke area — which increases the complexity.

And there's also some tubes that have LEDs internally that are drop in replacements for fluorescents.

## TL:DR

So our options are: 

 1. Replace the fluorescent tubes (most direct),

OR

 2. Replace the fluorescent tubes with LED tubes (drop in replacement), 

OR

 3. Rewire the light switch to low voltage LED strips (most work, but most flexible light positioning).

**Comment by DefProc** [comment]
I realised that I've just added information, without actually moving this forward at all. 

So: I've measured the tubes on both machines: 

 * Sophia: 550mm long, T5 tube (16mm diameter),
 * Gerald: 1147mm long, T5 tube (16mm diameter).

As a drop in replacement is quickest (and these tubes have been not working for *months*), I've ordered direct replacements for both (a few pounds each, compared to >£30 for LEDs), with should arrive tomorrow. This should allow more time to look at the best LED replacement, without holding up functionality (any longer).

*Interestingly, if you start looking for LED replacement for these size tubes, you end up on hydroponics supplier websites; just like for laser cutter extraction fans!*

**Comment by DefProc** [comment]
New tubes fitted. Is there still an issue to fit LED lighting, or is this a suitable solution for this issue?

**Comment by DefProc** [comment]
If there's no objection, I'll close this for now.
